runelite-client/src/main/java/net/runelite/client/plugins/fairyring/FairyRingConfig.java

```diff
@@ -34,10 +34,22 @@ public interface FairyRingConfig extends Config
 	@ConfigItem(
 		keyName = "autoOpen",
 		name = "Open search automatically",
-		description = "Open the search widget every time you enter a fairy ring"
+		description = "Open the search widget every time you enter a fairy ring",
+		position = 1
 	)
 	default boolean autoOpen()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "enableFairyTags",
+		name = "Enable Tag searching",
+		description = "Enable the ability to add tags to search fairy ring codes by",
+		position = 2
+	)
+	default boolean enableFairyTags()
```

Comment:
Personally, I think this can be removed in favor it always being enabled. Other people can provide input on this.